index.js

```diff
@@ -116,6 +116,27 @@ class RCTVoice {
       });
     });
   }
+
+  /**
+   * (Android) Get a list of the speech recognition engines available on the device
+   * */
+  getSpeechRecognitionServices() {
```

Comment:
can you add this to the docs?

also show an example of result of calling this method